src/components/AddPlayerForm.js

```diff
@@ -54,13 +61,13 @@ function AddPlayerForm(props) {
     };
 
         return (
-            <Form noValidate onSubmit={submitHandler} validated={enteredNameIsValid && enteredEmailIsValid}>
-                <Form.Group className="mb-3">
-                    <Form.Label htmlFor='playername'>Name</Form.Label>
+            <Form noValidate onSubmit={submitHandler} validated={enteredNameIsValid && enteredEmailIsValid} className='d-grid gap-2'>
```

Comment:
Out of curiosity, what is `d-grid gap-2` class?

Reply:
To make the button size be adjusted with Form group, this makes full-width, according to [react-bootstrap](https://react-bootstrap.netlify.app/components/buttons/#buttons).

Reply:
Cool. Thx 👍